src/values.js

```diff
@@ -216,10 +217,41 @@ FaunaDate.prototype.toJSON = function()  {
   return { '@date': this.value };
 };
 
+/** FaunaDB bytes. See the [docs](https://fauna.com/documentation/queries#values-special_types).
+ *
+ * @param {Uint8Array|ArrayBuffer|string} value
+ *    If ArrayBuffer it's converted to Uint8Array
+ *    If string it must be base64 encoded and it's converted to Uint8Array
+ * @extends module:values~Value
+ * @constructor
+ */
+function Bytes(value) {
+  if (value instanceof ArrayBuffer) {
+    this.value = new Uint8Array(value);
+  } else if (typeof value === 'string') {
+    this.value = base64.toByteArray(value);
+  } else {
```

Comment:
Since this is a public interface, it would be safer to if we throw an exception when people try to create a `Bytes` instance without a valid binary type.